qiskit/circuit/library/data_preparation/pauli_feature_map.py

```diff
@@ -222,7 +223,8 @@ def z_feature_map(
 
         >>> from qiskit.circuit.library import TwoLocal
         >>> ry = TwoLocal(3, "ry", "cz", reps=1).decompose()
-        >>> classifier = z_feature_map(3, reps=1) + ry
+        >>> classifier = z_feature_map(3, reps=1)
+        >>> classifier.append(ry, list(range(classifier.num_qubits)))
```

Comment:
This is a good catch! Would you mind updating the snippet to use the new `n_local` function instead of the `TwoLocal` class?